modules/update-service-overview.adoc

```diff
@@ -48,5 +48,18 @@ The OpenShift Update Service is composed of an Operator and one or more applicat
 [NOTE]
 ====
 During the upgrade process, nodes in the cluster might become temporarily unavailable. The `MachineHealthCheck` might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, remove any `MachineHealthCheck` resource that you have deployed before updating the cluster.
-However, a MachineHealthCheck resource that is deployed by default (such as `machine-api-termination-handler`) cannot be removed and will be recreated.
+However, a `MachineHealthCheck` resource that is deployed by default (such as `machine-api-termination-handler`) cannot be removed and will be recreated.
```

Comment:
I'm wondering what happens when a default MHC gets paused... I'd expect it works, but needs to be tested to be sure

Reply:
Sure. Do let me know if pausing works on the default MHC. I will update this note accordingly.

Reply:
Pausing default MHC should work AFAIK.

We should be removing the note above here that talks about removing the health checks, that is not the recommendation we want to make. User should be using the pause mechanism instead of removing the resource

Reply:
Thank you @slintes and @JoelSpeed. I have updated the content to remove reference to removing the MHC in the context of updating the cluster. Kindly review.